### PR TITLE
Fix pre-commit checks

### PR DESCRIPTION
**Pull request by reakaleek**
## What does this pull request do?

Currently, sanity-checks are failing on the main branch because `pre-commit` is run on all files on the main branch. (Source: https://github.com/elastic/apm-agent-ruby/actions/workflows/ci.yml?query=branch%3Amain)

This PR fixes sanity-checks on the main branch. 

## Why is it important?

[CI](https://github.com/elastic/apm-agent-ruby/actions/workflows/ci.yml?query=branch%3Amain) checks are red on the main branch.

### CODE REVIEW

File: spec/fixtures/w3c_distributed_tracing.json
Comment (reakaleek) [inline spec/fixtures/w3c_distributed_tracing.json:1]: Suggestions, where to store this information instead?

File: spec/fixtures/w3c_distributed_tracing.json [hunk 1]
```diff
@@ -1,11 +1,3 @@
-//
-// from https://github.com/w3c/distributed-tracing/blob/main/test/test_data.json
-//
-// NOTE: This file is manually copied from the above link and
-//       there is currently NO automation keeping it in sync with the upstream version.
-//
-// Latest commit 98f210e on Sep 24, 2019
-//
```
Comment (reakaleek) [inline spec/fixtures/w3c_distributed_tracing.json:8]: Moved the comment to the commit message.

image

Comment (reakaleek) [reply spec/fixtures/w3c_distributed_tracing.json:8]: Oh, wait. This information is lost when merging with squashing.

Any suggestions where to store this information instead?

**Review by v1v — APPROVED** [review]

**Review by estolfo — APPROVED** [review]

**Comment by reakaleek** [comment]
Unable to get all matrix items green after 7 attempts.

However, changes are unrelated to failing test. Hence, I'm merging this.

**Pull request merged**